initial page for large-scale-processing (#67)
openEOPlatform/documentation: usecases/large-scale-processing/index.md

specific tile. This gives us a very good visual overview of the processing that will be performed.

Having job parameters in a file is also useful for debugging afterwards. Determining parameters at runtime means you don't 
have absolute certainty over the value of a specific argument, as there may be bugs in your code.

## Prepare tiling grid

The choice of tiling grid depends on your preferred projection system, which depends on your area of interest. 
For Europe, the EPSG:3035 projection can be used, while for global processing you may want to work with different projections according
to UTM zones.

The size of tiles in your grid is also important, and often ranges from 20km to 100km. For relatively light workflows, a 100km grid can work well, 
while for more demanding cases, a 20km grid is better. In our example, we choose to work with 20km tiles because the workflow was quite demanding. A smaller 
tile size can also result in less unneeded processing when your target area has an irregular shape, like most countries and continents.

A couple of basic grids can be found here:

> https://artifactory.vgt.vito.be/webapp/#/artifacts/browse/tree/General/auxdata-public/grids

Reviewer: Make it a link?

```suggestion
<https://artifactory.vgt.vito.be/webapp/#/artifacts/browse/tree/General/auxdata-public/grids>
```

available to finish your job in time. This can be done by simply running various jobs, and using the statistics reported in the metadata to determine average 
parameters.

For instance, for the case of processing the EU27 croptype map, consisting of ~11000 20km tiles, we made the following calculations op front:

- Average runtime was 30 minutes, which means that it would take ~15 days of continous processing with 15 parallel jobs.
- Average cost was below 100 credits, so we would be able to process with a budget of 1100000 credits.

To achieve these numbers, we did have to optimize batch job settings and also the overall workflow to reduce resource usage!

A common bottleneck to parallellization is the memory consumption, and it can be useful to know the maximum memory allocation on a single machine in
your backend of choice. For instance, in a cloud environment with 16GB per machine and 4 cpu's, using slightly less than 4GB per worker is efficient as you can fit 4 parallel workers on a single VM, while requiring 6GB would fit only 2 workers and leave about 4GB unused.

> In our example, we used the Geotrellis backends, which has [these tunables](../../federation/#customizing-batch-job-resources-on-terrascope).

Reviewer: Im love the term tunables, although I guess it doesn't exist. Maybe "execution options" or somethign similar is easier to understand?

Anyway, my main point here is that the link should be different though:

```suggestion
In our example, we used the Geotrellis backends, which has [these execution options](../../federation/index.md#customizing-batch-job-resources-on-terrascope).
```

## Starting map production

The openEO Python client provides a useful tool to run multiple processing jobs in multiple backends:

> https://open-eo.github.io/openeo-python-client/cookbook/job_manager.html

Reviewer: ```suggestion
<https://open-eo.github.io/openeo-python-client/cookbook/job_manager.html>
```

make it a link
